packages/jellyfish-api-core/__tests__/category/rawtx/decodeRawTransaction.test.ts

```diff
@@ -0,0 +1,101 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers/dist/index'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { RpcApiError } from '../../../src'
+
+describe('poolpair update', () => {
+  const container = new MasterNodeRegTestContainer()
```

Comment:
```suggestion
  const container = new RegTestContainer()
```

Maybe we can just use this container?